app/src/main/java/com/it/hungvt/movieapp/screen/detail/DetailViewModel.java

```diff
@@ -0,0 +1,93 @@
+package com.it.hungvt.movieapp.screen.detail;
+
+import android.content.Context;
+import android.content.Intent;
```

Comment:
ERROR: Unused import: android.content.Intent. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports